padertorch/base.py

```diff
@@ -253,6 +256,9 @@ def review(self, inputs, outputs):
          - The contextmanager `torch.no_grad()` disables backpropagation for
            metric computations (i.e. scalars in tensorboard)
          - `self.training` (bool) indicate training or validation mode
+         - `self.create_snapshot` (bool) indicates when to create a snapshot.
+            Useful for reporting of things that need many computational
+            resources (e.g., istft)
```

Comment:
Maybe highlighting more, that only one snapshot is stored from the summary hook?
e.g.
> `self.create_snapshot` (bool) indicates when to create a snapshot.
> Motivation: Only one snapshot is reported from the summary hook for snapshot types (e.g. images, audios, ...) for each summary interval. You can use this flag to avoid unnecessary computations and compute only one snapshot that will be reported instead of computing many snapshot that will later be deleted.

Reply:
Done